addons/binding/org.openhab.binding.hp1000/README.md

```diff
@@ -0,0 +1,59 @@
+# org.openhab.binding.hp1000
+
+HP100X/WH260X weather stations binding for OpenHab witch supports the personal weather station upload protocol.
+
+# Thing Configuration
+
+Automatic discovery is not supported. You have to do a manual configuration.
+
+## Configuration Parameters
```

Comment:
Please provide a note that you binding does not have configuration parameters instead of leaving this section empty.